### DISCUSSION TITLE
Ipmitool - imb does not work on Windows

### DISCUSSION THREAD

**Discussion by zbyhuu** [ipmitool/ipmitool#313] [Q&A]
Hi All,
I tried to compile ipmitool for Windows using cygwin.
Followed the instruction in INSTALL file.
Added --enable-intf-imb DEFAULT_INTF="imb" to ./configure.
Ipmitool successfully compiled but imb doesn't work:
"Error: no IMB driver found at /dev/imb!"

Intel Intelligent Management Bus Driver V13.0 is installed on my system and this particular unit has a BMC.
When I connect to the BMC by -I lan it works perfeclty, however I need sometimes use KCS to get the interested data from BMC via ipmi.

Have any of you encountered such a problem?

Best regards,
Zbigniew

**Comment by doublechiang** [comment]
I recall windows driver is not supported, you can only use the LAN interface on Windows.

**Reply by zbyhuu** [reply]
Okee, got it, 
Could you point me out starting from which version?

Thank you!

**Comment by doublechiang** [comment]
I think windows has its own IPMI driver. Linux uses ipmi_msghandler driver.
As far as I understand, this project never supports windows in-band operation.

**Comment by AlexanderAmelkin** [comment]
considering that IMB is purely a windows thing, and from the error message you get (regarding /dev/imb that can't exist natively on Windows), I can only guess that there must be some Cygwin magic involved that would add IMB driver as /dev/imb.

Unfortunately, I'm not proficient with Cygwin or Windows and so can not help any further.
